extensions/3DTILES_metadata/README.md

```diff
@@ -32,519 +39,522 @@ This extension is optional, meaning it should be placed in the tileset JSON `ext
+* **Tile** - Tiles may be individually associated with more specific metadata, such as the timestamp when a tile was last updated, or maximum height of the tile's content.
+* **Tile Content Groups** - Tile contents may be organized into collections (see: [Groups](#content-group-properties)) with shared metadata.
 
-* Tileset metadata - metadata about the tileset as a whole.
-* Tile metadata - metadata about individual tiles.
-* Group metadata - metadata about groups of content.
-* Feature metadata - metadata about features. See the companion glTF extension [EXT_mesh_features](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features).
+> **Implementation note:** Certain subcomponents of tile content ("features") may also have associated metadata. See [Content Feature Properties](#content-feature-properties).
+
+Concepts and terminology used throughout this document refer to the [Cesium 3D Metadata Specification](../../specification/Metadata/README.md), which should be considered a normative reference for definitions and requirements. This document provides inline definitions of terms where appropriate.
+
+The figure below shows the relationship between entities (tilesets, tiles, contents, and groups) in 3D Tiles:
 
 <img src="figures/metadata-granularity.png"  alt="Metadata Granularity" width="600">
 
-A tileset defines a **schema**. A schema has a set of **classes** and **enums**. A class contains a set of **properties**, which may be numeric, boolean, string, enum, array, vector, or matrix types.
+## Use Cases
+
+_This section is non-normative_
 
-**Entities** (such as tiles, features, etc.) conform to classes and contain **property values**. Depending on the context, property values may be stored in JSON or binary.
+Metadata in 3D Tiles enables additional use cases and functionality for the format:
 
-**Statistics** provide aggregate information about the metadata. For example, statistics may include the min/max values of a numeric property for mapping property values to color ramps with the [declarative styling language](../../specification/Styling) or the number of enum occurrences for creating histograms.
+- **Inspection:** Applications displaying a tileset within a user interface (UI) may allow users to click or hover over specific tiles or tile contents, showing informative metadata about a selected entity in the UI.
+- **Collections:** Tile content groups may be used to define collections (similar to map layers), such that each collection may be shown, hidden, or visually styled with effects synchronized across many tiles.
+- **Structured Data:** Metadata supports both embedded and externally-referenced schemas, such that tileset authors may define new data models for common domains (e.g. for AEC or scientific datasets) or fully customized, application-specific data (e.g. for a particular video game).
```

Comment:
hm... reading this again on a new day, I think we might want to reword this again.

1. I don't think we should abbreviate AEC here, the reader might not understand that this is architecture, engineering and construction. I know the first time you used this in an issue I had to look it up :sweat_smile:. Though it is long when written out. If this bullet needs to be split into a few smaller sentences, that's probably okay
2. maybe make it "for AEC or scientific **applications**". 
3. I feel like this paragraph should mention the word "interoperability" somewhere.